packages/plugin-css/src/utils/is-token-match.ts

```diff
@@ -0,0 +1,13 @@
+import wcmatch from '../wildcard-match.js';
+
+/** match token against globs */
+export function isTokenMatch(tokenID: string, globPatterns: string[]): boolean {
```

Comment:
Appreciate the extra attention to detail in cleaning up the utilities, and adding tests.